tests/features/communities/tallinn/last_update.feature

```diff
@@ -0,0 +1,30 @@
+@api @tallinn
+Feature:
+  As a moderator of Joinup
+  I want to be able to be aware when was the last update time of a tallinn report.
+
+  Background:
+    Given users:
+      | Username | Roles     |
+      | karma    | moderator |
+
+  Scenario: Tallinn reports that have never been updated should not show last update date.
+    # Timestamp 1530529671 translates to 02-07-2018.
+    Given tallinn_report content:
+      | title          | collection                      | created    | changed    |
+      | Romania Report | Tallinn Ministerial Declaration | 1530529671 | 1530529671 |
```

Comment:
Can't we use human readable dates? It's done in other places like tests/features/community_content/add_community_content.feature:79 .